From: Andriy Grytsenko <andrej@rep.kiev.ua>
Date: Thu, 14 Sep 2017 14:25:15 +0300
Subject: _no_gettext

# Don't use both gettext and intltool since newer versions of intltool
# conflict with gettext
---
 configure.in | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/configure.in b/configure.in
index 6335ed3..3c4278b 100644
--- a/configure.in
+++ b/configure.in
@@ -158,11 +158,9 @@ dnl ========================================================
 dnl Translations
 dnl ========================================================
 
-AM_GNU_GETTEXT([external])
-AM_GNU_GETTEXT_VERSION([0.17])
 GETTEXT_PACKAGE=gnome-system-tools
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [gettext package name])
-AC_SUBST([GETTEXT_PACKAGE], GETTEXT_PACKAGE)
+AC_SUBST([GETTEXT_PACKAGE])
 
 dnl ========================================================
 dnl END : Translations
